### PR TITLE
overlay min-turnup into kubernetes anywhere

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

File: kubeconfig.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        "server": "https://104.154.39.72"
```
Comment (errordeveloper) [inline kubeconfig.json]: Was this intentional?

File: phase2/ansible/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM debian:jessie
+
+RUN apt-get update && apt-get install -y git python-dev python-setuptools build-essential curl vim jq libffi-dev libssl-dev
+
+RUN easy_install pip
+RUN pip install paramiko PyYAML Jinja2 httplib2 six pycrypto
+
+RUN git clone git://github.com/ansible/ansible.git --recursive /opt/ansible
```
Comment (errordeveloper) [inline phase2/ansible/Dockerfile]: It'd be good to checkout release branch or tag. If we want latest, may be could checkout a specific revision?

File: phase2/ansible/playbooks/roles/node/templates/kubelet.service.j2 [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet
+ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
+ExecStart=/usr/bin/docker run \
+        --net=host \
+        --pid=host \
+        --privileged \
+        -v /sys:/sys:ro \
+        -v /var/run:/var/run:rw \
+        -v /var/lib/docker/:/var/lib/docker:rw \
+        -v /var/lib/kubelet/:/var/lib/kubelet:shared \
```
Comment (errordeveloper) [inline phase2/ansible/playbooks/roles/node/templates/kubelet.service.j2]: Is `shared` enough? I though this has to be `rshared`...

Comment (errordeveloper) [inline phase2/ansible/playbooks/roles/node/templates/kubelet.service.j2]: Also, may be add `rw` as well, so it's more consistent with the rest.

**Comment by errordeveloper** [comment]
I had a few thoughts about `phase2/ansible/Dockerfile`, e.g. purging `build-essential` and checking out specific revs for ansible and jsonnet git repos, but may be we don't have to do these things right now. Overall - LGTM.

File: kubeconfig.json (hunk 1 above)
Comment (mikedanese) [inline kubeconfig.json]: Oops. Security breach.

File: phase2/ansible/playbooks/roles/node/templates/kubelet.service.j2 (hunk 3 above)
Comment (mikedanese) [inline phase2/ansible/playbooks/roles/node/templates/kubelet.service.j2]: rshared is sufficient, added rw.

**Comment by mikedanese** [comment]
I agree that the Dockerfiles are currently very crude and could be trimmed.

**Pull request merged**